### PR TITLE
Meeting structure

### PR DESCRIPTION
**Pull request by urielShtainfeld**
add page numbers to qoutes, and add href for .pdf files after clicking qoute

### CODE REVIEW

File: our-brothers-web/src/app/agenda/page/agenda-page.component.html [hunk 1]
```diff
@@ -115,7 +115,12 @@ <h2 class="medium-heading" data-aos="fade-up" data-aos-duration="1000">
         <ng-container *ngFor="let quote of page.page" data-aos="fade-up" data-aos-duration="1000">
           <div class="quote">
             <img class="apostrophy" src="../../../assets/img/apostrophy.png" />
-            <div class="site-text quote-text">{{ quote.excerpt }}</div>
+            <a
+              class="site-text quote-text"
+              target="_blank"
+              [href]="'../../../assets/texts/' + quote.writerId + '.pdf'"
```
Comment (gilhanan) [inline our-brothers-web/src/app/agenda/page/agenda-page.component.html]: Can we remove file type?

File: our-brothers-web/src/app/agenda/page/agenda-page.component.scss [hunk 2]
```diff
@@ -141,4 +143,31 @@
       }
     }
   }
+
+  .pages-number-container {
```
Comment (gilhanan) [inline our-brothers-web/src/app/agenda/page/agenda-page.component.scss]: Can we remove the default `background: transparent;`?

File: our-brothers-web/src/app/agenda/page/agenda-page.component.html (hunk 1 above)
Comment (urielShtainfeld) [reply our-brothers-web/src/app/agenda/page/agenda-page.component.html]: I checked it out and it's won't work without file extension

File: our-brothers-web/src/app/agenda/page/agenda-page.component.scss (hunk 2 above)
Comment (urielShtainfeld) [reply our-brothers-web/src/app/agenda/page/agenda-page.component.scss]: I tried it, and the background is changed to grey.
so we must leave it here

File: our-brothers-web/src/app/agenda/page/agenda-page.component.html (hunk 1 above)
Comment (gilhanan) [reply our-brothers-web/src/app/agenda/page/agenda-page.component.html:121]: Make sure you use the file extension inside the `quote.writerId`. Also, consider to rename it.

**Pull request merged**